src/index.html

```diff
@@ -5,12 +5,51 @@
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Searchbar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    class="searchForm"
```

Comment:
Check the code style everywhere 
```suggestion
    <form 
      class="searchForm"
```
![image](https://user-images.githubusercontent.com/99403198/235617752-5c8c7dd4-d497-4bd3-8c9e-4199437cc537.png)